test/e2e/kubelet.go

```diff
@@ -145,6 +142,26 @@ func createNfsServerPod(c clientset.Interface, config VolumeTestConfig) (*v1.Pod
 	return pod, ip
 }
 
+// Restart the passed-in nfs-server by issuing a `/usr/sbin/rpc.nfsd 1` command in the
+// pod's (only) container. This command changes the number of nfs server threads from
+// (presumably) zero back to 1, and therefore allows nfs to open connections again.
+func restartNfsServer(serverPod *v1.Pod) {
+
+	const startcmd = "/usr/sbin/rpc.nfsd 1"
```

Comment:
@jingxu97 It starts the nfs-server pod and a client pod that will read/write to a nfs volume. Then the nfs-server is stopped and the client pod is deleted. Then the nfs-server is restarted. The test checks that the client pod's nfs mount has been deleted.